tests/e2e/test.sh

```diff
@@ -29,6 +29,39 @@ function selenium_start {
     nohup npm run selenium-start > ./tests/reports/selenium.log 2>&1&
 }
 
+function modify_config {
+    message "Modifying config..."
+
+    cd ../mattermost-server
+
+    echo "config: enable email invitation"
+    sed -i'' -e 's|"EnableEmailInvitations": false|"EnableEmailInvitations": true|g' config/config.json
+
+    echo "config: enable email notification"
+    sed -i'' -e 's|"SendEmailNotifications": false|"SendEmailNotifications": true|g' config/config.json
+
+    echo "config: enable mobile push notification"
+    sed -i'' -e 's|"SendPushNotifications": false|"SendPushNotifications": true|g' config/config.json
+    sed -i'' -e 's|"PushNotificationServer": ".*"|"PushNotificationServer": "https://push.mattermost.com"|g' config/config.json
+    sed -i'' -e 's|"PushNotificationContents": ".*"|"PushNotificationContents": "generic"|g' config/config.json
+
+    cd ../mattermost-webapp
+    sleep 5
+}
+
+function run_server {
```

Comment:
this is still a restart_server, not a big deal, but I prefer the old name.

Reply:
I thought the new name fits it but I think you're right.  Done update.  Thanks!